conrad-blucher-institute/Flare
# Development of the User Interface

I ran into an issue where Highcharts stopped fetching data when making the app mobile-friendly. To resolve this, I created separate chart pages for mobile and desktop.
## Testing Instructions

1. Update the nginx config
  - server_name is localhost
  - port is 80
  - path to file: /user/share/nginx/html
  - Note: I meant to just add this to the nginx template, but git isn't recognizing that I have made changes at the moment.
 2. Generate Test Data
    - You can generate test CSV data by running this command: python3 backend/tests/test_csv_generator.py
3. Open two WSL terminals side by side.
     - In one terminal, start the Docker containers.
     - In the other terminal, run the development server.
  
4. Once in the development server click the Turtle button to display the chart.
5. Verify that the chart graphs as expected.
## Notes:
- Gaps in the Chart: Any gaps in the chart are intentional and caused by the 10-hour interval in the test data, which was set to maintain a clean and uncluttered visualization during testing.
